### ISSUE TITLE
Changes to the content of the overview.csv files

### ISSUE THREAD

**Issue by bzorn** [microsoft/promptpex#32]
Here are the columns I'd like to see in the <benchmark>-overview.csv file

model, promptpex tests, promptpex tests compliant, % promptpex tests compliant, benchmark tests, benchmark tests compliant, % benchmark tests compliant, positive tests, positive tests compliant, % positive tests compliant, negative tests, negative tests compliant, % negative tests compliant, valid tests, valid tests compliant, % valid tests compliant, invalid tests, invalid tests compliant, % invalid tests compliant

Basically, the columns are arranged in groups of 3: number of tests, number of compliant tests, and % of compliant tests (= number of compliant tests / number of tests).

If we have these columns, I think that converting this into different charts/tables should be relatively easy.
